places-database-management-1/bulk-import-of-pois.md

```diff
@@ -42,18 +42,18 @@ The values of the following columns are used in the Location Service UI:
   - The valid values are "", #3E76D0, #AA99E8, #DC2ABA, #FC685B, #FC962E, #F6C436, #BECE5D, #61B56B, and #3DC8DE.
   - If the value is left blank, the Location Service UI uses blue as the default color.
     
-    The values correspond to blue, purple, fuschia, orange, light orange, yellow, light green, green, and light blue, respectively.
+    The values correspond to blue ( #3E76D0), purple (#AA99E8), fuschia (#DC2ABA), orange (#FC685B), light orange (#FC962E), yellow (#F6C436), light green (#BECE5D), green (#61B56B), and light blue (#3DC8DE), respectively.
   
 * icon, which is used as the icon on the pin that represents the location of the POI on the Location Service UI map
-  - The valid values are "", anchor, beaker, bell, browse, book, brush, building, calculator, camera, shoppingCart, clock, box, flashlight, follow, bid, ribbon, education, hammer, heart, home, key, mailbox, male, promote, money, trap, game, gift, launch, star, lightbulb, pin, target, teapot, thumbDown, thumbUp, briefcase, trophy, female, and wrench.
-  - If the value is left blank, the UI uses star as the default icon.
+  - The valid values are "", shop, hotelbed, car, airplane, train, ship, stadium, amusementpark, anchor, beaker, bell, bid, book, box, briefcase, browse, brush, building, calculator, camera, clock, education, flashlight, follow, game, female, male, gift, hammer, heart, home, key, launch, lightbulb, mailbox, money, pin, promote, ribbon, shoppingCart, star, target, teapot, thumbDown, thumbUp, trap, trophy, wrench.
+  - If the value is left blank, the UI uses heart as the default icon.
 
 * Columns that are not mentioned can be left blank.
 
 
 Running the Script
 -------------
-1. Download files to the appropriate directory.
+1. Download files from the [git repo](https://github.com/adobe/places-scripts) to the desired directory.
```

Comment:
- Please remove "desired".
- They would only download the files to a directory right? So please reword as follows:
"Download files from the [git repo](https://github.com/adobe/places-scripts) to your local directory."